packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -210,6 +210,9 @@ const typeStringIos = async ( driver, element, str, clear ) => {
 		await clearTextBox( driver, element );
 	}
 	await element.type( str );
+
+	// Wait for the list auto-scroll animation to finish
+	await driver.sleep( 3000 );
```

Comment:
Yeah, it will increase the run time a bit. I'll monitor that as well as if it reduces flakiness on some of the tests. We might be able to reduce this timer or just use it in some specific cases.